### DISCUSSION TITLE
deploy moclojer on clojars

### DISCUSSION THREAD

**Discussion by matheusfrancisco** [moclojer/moclojer#180] [Ideas]
I was thinking of deploying the moclojer on clojars.
Let's say I have a microservice and I want to have an api up when I run some tests I can create some specs and start Moclojer after my test runs. 

What do you think?

so if we added moclojer to clojars would be easier to use it on microservices, right?

**Comment by avelino** [comment]
we will distribute moclojer in clojars, but first we need to organize our consumption APIs (namespaces) so that they can be consumed in a "friendly" way by external applications.

I believe that this other discussion https://github.com/moclojer/moclojer/discussions/170 is linked to this subject, even though the title contains "javascript", I'm talking about the clojure api


> closing this discussion to centralize on #170, avoiding two parallel discussions - updated the main discussion talking about distribution on clojars
